docs/installation/quick.md

```diff
@@ -93,6 +97,13 @@ $ leo_storage/bin/leo_storage start
 $ leo_gateway/bin/leo_gateway start
 ```
 
+For systemd-based distributions running v1.4.0 or later, you can use systemd units to launch LeoFS instead:
+```text
+# systemctl start leofs-manager-master leofs-manager-slave leofs-gateway leofs-storage
+```
```

Comment:
To get the special attention from readers, please use ```!!! note Do not mix launching/stopping methods``` a special syntax that can be used on mkdocs. 
An example can be seen
- https://mocchira.github.io/leofs/admin/system_operations/s3/#create-a-bucket-add-bucket
  Note: Creating bucket with the same name as one being removed